src/main.js

```diff
@@ -30,14 +40,25 @@ if (args.dev) {
     process.env.NODE_ENV = "production";
 }
 
-// Add logging support, based on
-//   https://cloud.google.com/nodejs/getting-started/logging-application-events
-const appWithLogging = express();
-
-appWithLogging.use(middleware.requestLogger);
-appWithLogging.use(middleware.errorLogger);
-appWithLogging.use(app);
+/**
+ * Create the express app.
+ *
+ * Logging support is based on:
+ *   https://cloud.google.com/nodejs/getting-started/logging-application-events
+ *
+ * The request logger should come before the app, and the error logger, after.
+ */
+const appWithLogging = express()
+    .use(middleware.requestLogger)
+    .use(app)
+    .use(middleware.errorLogger);
```

Comment:
Use the chaining ability of express to tidy up a but. Also added lots more commentary in here and reordered the middleware to match what express-winston says it should be.